mono/mini/aot-runtime-wasm.c

```diff
@@ -120,7 +90,7 @@ get_long_arg (InterpMethodArguments *margs, int idx)
 
 #include "wasm_m2n_invoke.g.h"
 
-static void
+void
 wasm_interp_to_native_trampoline (void *target_func, InterpMethodArguments *margs)
```

Comment:
Since `static` is removed, this must be prefixed with `mini_` or `mono_`.

Reply:
Fixed.